packages/hooks/src/useMT5SVGEligibleToMigrate.ts

```diff
@@ -0,0 +1,65 @@
+import { useMT5AccountsList } from '@deriv/api';
+import { Jurisdiction, getFormattedJurisdictionCode } from '@deriv/shared';
+
+const useMT5SVGEligibleToMigrate = () => {
+    const { data: mt5_login_list = [] } = useMT5AccountsList();
```

Comment:
a doubt, after this point, would it be any better if we write everything inside memo?
`React.memo( () => {.. all logic here and return result ..}, [ mt5_login_list ] );`